UnityProject/Assets/Scripts/Core/Chat/Chat.Process.cs

```diff
@@ -38,13 +38,14 @@ private struct DestroyChatMessage
 	public Color localColor;
 	public Color combatColor;
 	public Color warningColor;
+	public Color blobColor;
 	public Color defaultColor;
 
 
 	/// <summary>
 	/// This channels can't be heared as sound by other players (like binary or changeling hivemind)
 	/// </summary>
-	public const ChatChannel NonVerbalChannels = ChatChannel.Binary | ChatChannel.Ghost;
+	public const ChatChannel NonVerbalChannels = ChatChannel.Binary | ChatChannel.Ghost | ChatChannel.Blob;
```

Comment:
Codacy found an issue: [Change this constant to a 'static' read-only property.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6358537)